ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less

```diff
@@ -0,0 +1,236 @@
+/* tabs style */
+
+
+.tabs{
+        
+
+
+    .get_type1_tab(@color,@bgcolor) {
```

Comment:
@Rupesh-Shende @rakeshkumar-verma : the foct icon is white on grey backgroud , please update as its not visible based on active class.

![image](https://user-images.githubusercontent.com/38690824/81678443-359d3f80-946f-11ea-832c-0f96c6110252.png)

Reply:
@rishabh-behl Icon color updated as per UX design.
![image](https://user-images.githubusercontent.com/63295621/81720133-9a6e8f00-949b-11ea-8d32-062fb25cddb8.png)